src/sentry/static/sentry/app/views/settings/project/projectReleaseTracking.jsx

```diff
@@ -23,8 +23,8 @@ const noMargin = {margin: 0};
 const marginTop = {marginTop: 30};
 
 const PreWrap = styled.pre`
-  word-break: break-all;
-  white-space: pre-wrap;
+  word-break: break-all !important;
```

Comment:
that was my guess lol.

This will make any attempts to override these lines up the tree extremely difficult. Can we instead do:

```jsx
const PreWrap = styled.pre`
  pre& {
    word-break: break-all;
    white-space: pre-wrap;
    word-break: break-all !important;
  }
`
```